README.md

```diff
@@ -9,29 +9,56 @@ This command-line tool written in Java that performs the following steps:
 1. Checks file integrity, numeric type parsing and ranges as well as string format according to the [GTFS Schedule specification](https://gtfs.mobilitydata.org/spec/gtfs-schedule#h.hc443y62gb8c)
 1. Performs GTFS business rule validation *(work-in-progress)*
 
-# Build the code
-We suggest using [IntelliJ](https://www.jetbrains.com/idea/download/) to [import](https://www.jetbrains.com/help/idea/import-project-or-module-wizard.html), build, and run this project.
+# Run the app via command line
+### Setup
+1. Install [Java 8 or higher](https://www.oracle.com/java/technologies/javase-downloads.html)
+1. Download [gtfs-validator-v2.0.0_cli.jar](https://github.com/MobilityData/gtfs-validator/releases/download/v2.0.0/gtfs-validator-v2.0.0_cli.jar)
```

Comment:
Add "Prepare your input directory. It has to be a compressed .zip folder containing all the GTFS feed text files. 
To compress a folder correctly, select all text files and select "Compress". Do not compress a folder from the folder icon."